src/bci_build/package/__init__.py

```diff
@@ -132,10 +132,10 @@ class OsVersion(enum.Enum):
     SP4 = 4
     #: SLE 15 Service Pack 3
     SP3 = 3
+    #: SUSE Linux Framework One
+    SLE16_0 = 16
```

Comment:
can't you use "16.0" here? this way you can use `enum.value` similar to Tumbleweed.

Reply:
we can. I don't know exactly how the basalt/sle sp/component architecture model story is turning out, so I'm a bit conservative with assumptions on numbers. I can change it, I don't think it matters right now one way or the other.

Reply:
We already return `16.0` from `pretty_print` and `OsContainer.os_version`, so imho it would be a bit more consistent to use 16.0 here.